README.md

```diff
@@ -4,6 +4,21 @@
 
 This repository contains documentation for Kuadrant, built using MkDocs and the `mike` plugin for multi-versioning. You can run and build these docs using Docker/Podman or by installing MkDocs locally.
 
+## Contributing
+
+The site configuration and navigation layout are defined in [./mkdocs.yml](./mkdocs.yml).
+The navigation sections follow the [Diátaxis framework](https://diataxis.fr/), with four main areas:
+
+- Concepts (known as 'Explanation' in Diátaxis)
+- APIs & Reference
+- Tutorials
+- Guides
+
+When adding new documentation, take a moment to consider which area the document belongs to.
```

Comment:
Maybe add a note to mention what do to do if unsure (create a PR anyway, raise an issue to ask, or ask on #kuadrant?)

Reply:
Added.
Also, moved this block out to a [CONTRIBUTING.md](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) file